content/en/blog/_posts/2023-08-15-retroactive-default-storage-class-ga.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: blog
+title: "Kubernetes v1.28: Retroactive Default StorageClass move to GA"
+date: 2023-08-15
```

Comment:
Hi @RomanBednar , here Comms shadow, could you please edit for the scheduled date?
```suggestion
date: 2023-08-18
```

Reply:
@krol3 Done and squashed.